docs/podman-generate-systemd.1.md

```diff
@@ -28,41 +31,66 @@ Set the systemd restart policy.  The restart-policy must be one of: "no", "on-su
 "on-watchdog", "on-abort", or "always".  The default policy is *on-failure*.
 
 ## Examples
-Create a systemd unit file for a container running nginx:
 
+Create a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout on stdout.
 ```
-$ sudo podman generate systemd nginx
+$ podman create --name nginx nginx:latest
+$ podman generate systemd --restart-policy=always -t 1 nginx
+# container-de1e3223b1b888bc02d0962dd6cb5855eb00734061013ffdd3479d225abacdc6.service
+# autogenerated by Podman 1.5.2
+# Wed Aug 21 09:46:45 CEST 2019
+
 [Unit]
-Description=c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc Podman Container
+Description=Podman container-de1e3223b1b888bc02d0962dd6cb5855eb00734061013ffdd3479d225abacdc6.service
+Documentation=man:podman-generate-systemd(1)
+
 [Service]
-Restart=on-failure
-ExecStart=/usr/bin/podman start c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
-ExecStop=/usr/bin/podman stop -t 10 c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
+Restart=always
+ExecStart=/home/valentin/go/src/github.com/containers/libpod/bin/podman start de1e3223b1b888bc02d0962dd6cb5855eb00734061013ffdd3479d225abacdc6
```

Comment:
I don't think you should have your name in the man page.  Should be 
/bin/podman start de1e3223b1b888bc02d0962dd6cb5855eb00734061013ffdd3479d225abacdc6

Reply:
Thanks for checking!